printer_mods/watt-this/in-situ_linear_rail_greasing/.metadata.yml

```diff
@@ -0,0 +1,30 @@
+title: In Situ Linear Carriage Block Lubrication
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: A modification that allow lubrication of the linear reals while fully assembled.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: Jan_P
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- greasing_concept.step
```

Comment:
Files listed in the metadata need to include the path to that file, this should be:
` - CAD/greasing_concept.step`